web client/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="shortcut icon" href="static/images/favicon.ico" type="image/x-icon">
+    <link rel="stylesheet" href="static/css/adaptive.css">
+    <meta charset="UTF-8">
+    <title>Ratelance</title>
+    <link rel="stylesheet" href="static/css/index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
```

Comment:
Is double preconnect really needed? Also, I somewhat doubt that preconnect to `fonts.googleapis.com` is needed since a style is imported from there directly below.

Reply:
I think I forgot to remove the preconnect when I changed the font, there was originally a different one. I didn't think about the necessity, to be honest, I guess it's really not that important